src/adminBase.js

```diff
@@ -152,6 +139,23 @@ export default class AdminBase extends HasModels {
       ? this.models.model.update(values.id, values)
       : this.models.model.create(values)
 
-    promise.then((u) => {req.flash('info', this.display_name()+' created');; res.redirect(req.adminOpts.basePath+this.base)})
+    promise.then((u) => {req.flash('info', this.opts.display_name+' created');; res.redirect(req.adminOpts.basePath+this.base)})
+  }
+
+  _getAttrs(model) {
+    let ignore = this.opts.ignore || ['id', 'createdAt', 'updatedAt']
+    let ignoreType = ['objectId']
```

Comment:
These default type lists should move to be attributes of AdminBase (for override)